<launch>
    <node pkg="pathplanning" type="run_node.py" name="triangulation" respawn="true" output="screen">
        <param name="output_frame" value="ugr/car_odom" />
        <param name="algorithm" value="tri" />
        <param name="max_iter" value="100" />
        <param name="triangulation_min_var" value="100" />
        <param name="triangulation_var_threshold" value="2" />
        <param name="max_angle_change" value="0.5" />
        <param name="max_path_distance" value="6" />
        <param name="safety_dist" value="0.5" />
        
        <remap from="/input/local_map" to="/ugr/car/observations/global"/>
        <remap from="/output/path" to="/pathplanning/path"/>
        <remap from="/output/path_stamped" to="/pathplanning/path_stamped"/>

        <param name="debug_visualisation" value="False" />
        <param name="vis_namespace" value="pathplanning_vis" />
        <param name="vis_lifetime" value="0.2" />
        <remap from="/output/debug/markers" to="/pathplanning/debug/markers"/>
        <remap from="/output/debug/poses" to="/pathplanning/debug/poses"/>
    </node>
</launch>
